migrations/2023-07-17-111427_add-fraud-check-table.sql/up.sql

```diff
@@ -0,0 +1,34 @@
+-- Your SQL goes here-- Your SQL goes here
+CREATE TYPE "FraudCheckType" AS ENUM (
+    'pre_frm',
+    'post_frm'
+);
+
+CREATE TYPE "FraudCheckStatus" AS ENUM (
+    'fraud',
+    'manual_review',
+    'pending',
+    'legit',
+    'transaction_failure'
+);
+
+CREATE TABLE fraud_check (
+    id SERIAL PRIMARY KEY,
+    frm_id VARCHAR(255) NOT NULL UNIQUE,
+    payment_id VARCHAR(255) NOT NULL,
+    merchant_id VARCHAR(255) NOT NULL,
+    attempt_id VARCHAR(255) NOT NULL UNIQUE,
+    created_at TIMESTAMP NOT NULL DEFAULT now(),
+    frm_name VARCHAR(255) NOT NULL,
+    frm_transaction_id VARCHAR(255) UNIQUE,
```

Comment:
This also can't be a unique value, remove unique constraints here

Reply:
this is the id sent by signifyd(frm provider)
this will be unique only